egui/src/widgets/button.rs

```diff
@@ -242,26 +242,34 @@ impl<'a> Widget for Checkbox<'a> {
         let total_extra = button_padding + vec2(icon_width + icon_spacing, 0.0) + button_padding;
```

Comment:
`icon_spacing` should only be applied if there is text

Reply:
I tried to implement this, but I'm unsure about the second button padding.
As far as I can tell, this is fine.